pkg/v1/providers/ytt/02_addons/cni/antrea/antrea_overlay.lib.yaml

```diff
@@ -41,5 +41,7 @@ spec:
       initContainers:
         #@overlay/match by=overlay.subset({"name":"install-cni"})
         - image: #@ "{}/{}:{}".format(antreaImageRepo, antreaImage.imagePath, antreaImage.tag)
+        #@overlay/match by=overlay.subset({"name":"antrea-agent-tweaker"})
```

Comment:
Shouldn't the image override come from kbld file in the package bundle? @blc1996 - Did we not deprecate imageRepo?

Reply:
It is for initial deploy with CRS.